platform/src/main/java/org/stellar/anchor/platform/configurator/DataConfigAdapter.java

```diff
@@ -149,6 +152,9 @@ void updateSpringEnv(ConfigMap config) throws InvalidConfigException {
           set("spring.flyway.user", SecretManager.getInstance().get(SECRET_DATA_USERNAME));
           set("spring.flyway.password", SecretManager.getInstance().get(SECRET_DATA_PASSWORD));
           set("spring.flyway.url", constructPostgressUrl(config));
+          set(
+              "spring.flyway.baseline-on-migrate",
+              config.getString("data.flyway_baseline_on_migrate").equalsIgnoreCase("true"));
```

Comment:
It should also be fixed here:

```suggestion
          boolean baselineOnMigrate =
              config.getString("data.flyway_baseline_on_migrate").equalsIgnoreCase("true");
          if (baselineOnMigrate) {
            set("spring.flyway.baseline-on-migrate", true);
            set("spring.flyway.baseline-version", "0");
          }
```